src/core/link-to-dfn.js

```diff
@@ -24,12 +26,13 @@ export function run(conf, doc, cb) {
         if (oldIsDfn && newIsDfn) {
           // Only complain if the user provides 2 <dfn>s
           // for the same term.
+          dfn.attr('id',`${title + count}`);
           pub(
             "error",
-            "Duplicate definition of '" +
+            `<a href="#${title + count}">Duplicate definition of '` +
               (dfn_for ? dfn_for + "/" : "") +
               title +
-              "'"
+              `'</a>`
```

Comment:
Do you think there might be some way for us to group these? Like: "
```
  Duplicate definitions of term "X": 1, 2.
```  

Where 1 and 2 would be linked. 

Note also that the message, and the links, should be in markdown... so, you can just write the following to create the link:

```Markdown
[title](#id)
```

Reply:
I was thinking of the same thing while writing code. Sure I'll do this as well as all the changes you have requested for once I come back from college. Now got to rush :running_man:

Reply:
Awesome! Have fun and speak again soon.

Reply:
Back from college and I have started my work. Will update you soon

Reply:
GIF of changes are as follows:
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/30361728/35857606-a5175396-0b5f-11e8-9d2c-f8c44920ec49.gif)